e2e-tests/monitoring/02-create-cluster.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: |
+    . ../functions
+    set -o errexit
+    set -o xtrace
+
+    get_cr | yq eval '.spec.pmm.enabled=true' - | kubectl apply -n "${NAMESPACE}" -f -
```

Comment:
CR to compare pls

Reply:
I couldn't understand this comment. Do you mean to compare the CR in assertion?

Reply:
Yeap, at least the target state. Since you've added it I`m resolving this